src/etc/inc/openvpn.attributes.php

```diff
@@ -19,173 +19,259 @@
  * limitations under the License.
  */
 
-global $username;
-
-$devname = getenv("dev");
-if (empty($devname)) {
```

Comment:
Why remove this check? It should still default to the OpenVPN group rules if it can't locate a device.

Reply:
Ah yes good point. I don't use the group interface and overlooked this. Will correct.